src/OrderForm.js

```diff
@@ -29,26 +23,25 @@ class OrderForm extends Component{
         ev.preventDefault();
         let o = this.state;
         let final = {   
-            orderNumber: o.orderNumber,
-            totalAmount: o.totalAmount*1,
+            orderNumber: this.props.order.orderNumber,
             status: o.status
         }
-        this.props.updateOrder(o.id, final)
-        .then(()=>this.setState({...this.state, errors: []}))
-        .catch(e=>this.setState({...this.state, errors: e.response.data.errors}));
+        this.props.updateOrder(this.props.order.id, final)
+            .then(()=>this.setState({...this.state, errors: []}))
```

Comment:
Do we need to set errors : [], since the state is default to have errors : []

Reply:
If there were previous errors updating then yes, we would need to clear the errors on success as we do not leave this page.